Correct descriptions of failed examples in valid language tag rule (#2194)
act-rules/act-rules.github.io: _rules/html-page-lang-valid-bf051a.md

#### Failed Example 1

> This `html` element has a `lang` attribute, but it is not a [known primary language tag][].

Reviewer: ```suggestion
This `html` element has a `lang` attribute, with no [known primary language tag][].
```
The value of the `lang` attribute is not (and does not need to be) a primary language tag, only the, well, primary tag needs to be.

```html
<html lang="em-US"></html>
```

#### Failed Example 2

> This `html` element has a `lang` attribute, but it is not a [known primary language tag][].

Reviewer: ```suggestion
This `html` element has a `lang` attribute, with no [known primary language tag][].
```
